runtime/protocol/http-client/common/test/aws/smithy/kotlin/runtime/http/interceptors/FlexibleChecksumsRequestInterceptorTest.kt

```diff
@@ -126,23 +133,6 @@ class FlexibleChecksumsRequestInterceptorTest {
         assertEquals(0, call.request.headers.getNumChecksumHeaders())
     }
 
-    @Test
-    fun itSetsChecksumHeaderViaExecutionContext() = runTest {
-        checksums.forEach { (checksumAlgorithmName, expectedChecksumValue) ->
-            val req = HttpRequestBuilder().apply {
-                body = HttpBody.fromBytes("<Foo>bar</Foo>".encodeToByteArray())
-            }
-
-            val op = newTestOperation<Unit, Unit>(req, Unit)
-            op.context[HttpOperationContext.ChecksumAlgorithm] = checksumAlgorithmName
-            op.interceptors.add(FlexibleChecksumsRequestInterceptor<Unit>())
-
-            op.roundTrip(client, Unit)
-            val call = op.context.attributes[HttpOperationContext.HttpCallList].first()
-            assertEquals(expectedChecksumValue, call.request.headers["x-amz-checksum-$checksumAlgorithmName"])
-        }
-    }
```

Comment:
**Question**: Do we still use `HttpOperationContext.ChecksumAlgorithm` or can it be deprecated?

Reply:
I think it's still used for MD5 checksums in the [httpChecksumRequiredTrait](https://smithy.io/2.0/spec/http-bindings.html#httpchecksumrequired-trait)

Reply:
Update on this, it's no longer used anywhere so we can deprecate/remove it